cypress/support/generateNewUser.js

```diff
@@ -0,0 +1,12 @@
+const faker = require("faker");
+
+function generateNewUser() {
+    const randomNumber = Math.random().toString().slice(2, 6);
+    const username = faker.name.firstName() + `${randomNumber}`;
+    const email = `${username}@gmail.com`.toLowerCase();
+    const password = `Qwer1234`;
+
+    return { username, email, password };
+}
+
+module.exports = {generateNewUser};
\ No newline at end of file
```

Comment:
There is no newline at end of file. It's a good practice to put a newline at the end of the file because it ends the file on its own line so that if another line is added, the previous line won’t show up as modified.